services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/message/dataaccess/HasAccessorRequirementUtil.java

```diff
@@ -0,0 +1,26 @@
+package org.sagebionetworks.repo.manager.message.dataaccess;
+
+import java.util.Set;
+
+import org.sagebionetworks.repo.model.AuthorizationConstants;
+import org.sagebionetworks.repo.model.GroupMembersDAO;
+import org.sagebionetworks.repo.model.HasAccessorRequirement;
+import org.sagebionetworks.repo.model.VerificationDAO;
+import org.sagebionetworks.util.ValidateArgument;
+
+public class HasAccessorRequirementUtil {
+
```

Comment:
The static utility pattern should be reserved for pure functional programming.  The core design principal of PLFM is inversion-of-control (IoC).  I should be able to ask if a set of users has met the requirements without knowledge of which DAOs are needed to answer the question.  This should be in the AuthorizationManager.